src/lib/device-logs/backends/redis.ts

```diff
@@ -76,19 +76,25 @@ export class RedisBackend implements DeviceLogsBackend {
 		const redisLogs = logs.map(this.toRedisLog, this);
 
 		const limit = ctx.retention_limit || 0;
-		const key = this.getKey(ctx);
+		const logKey = this.getKey(ctx);
```

Comment:
renamed key for clarity when dealing in the same block of code with another redis key